Add singularity note for offline use (#2152)
nf-core/tools: README.md

nextflow run /path/to/download/nf-core-rnaseq-dev/workflow/ --input mydata.csv --outdir results  # usual parameters here
```

> Note that you want to use the downloaded singularity images, you will still need to use `-profile singularity` or have it enabled in your config file.

Author: ```suggestion
> Note that if you downloaded singularity images, you will need to use `-profile singularity` or have it enabled in your config file.
```
